templates/jlink/jlinked-app.yaml

```diff
@@ -86,18 +86,18 @@ objects:
       dockerStrategy:
         from: 
           kind: ImageStreamTag
-          name: ubi9-openjdk-${JDK_VERSION}:${BUILDER_IMAGE_TAG}  # Refer README.md to create this ImageStream
+          name: openjdk-${JDK_VERSION}-jlink-tech-preview:1.18  # Refer README.md to create this ImageStream
```

Comment:
Please restore the template-variable to specify tag and not hard-code 1.18; this will become out-of-date once I sync `jlink-dev` with `ubi9` branch.